app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -3,17 +3,25 @@ class OtpVerificationController < DeviseController
     include TwoFactorAuthenticatable
 
     def show
-      @phone_number = decorated_user.masked_two_factor_phone_number
-      @code_value = current_user.direct_otp if FeatureManagement.prefill_otp_codes?
-      @delivery_method = params[:delivery_method]
+      assign_variables_for_otp_verification_show_view
     end
 
     def create
-      if current_user.authenticate_direct_otp(params[:code].strip)
+      result = OtpVerificationForm.new(current_user, form_params[:code].strip).submit
+
```

Comment:
question: why is there an extra newline here? I usually do newlines before/after multiline blocks and in a few other contexts but it seems like you are following a rule of thumb that I am perhaps not familiar with